keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -1078,6 +1104,9 @@ Setting `Suspend:True` in your JobSpec will halt the execution of all jobs.
 - 2023-05-19: KEP Merged.
 - 2023-07-16: Alpha PRs merged.
 - 2023-09-29: KEP marked for beta promotion.
+- 2023-10-24: Merged bugfix [Fix tracking of terminating Pods when nothing else changes](https://github.com/kubernetes/kubernetes/pull/121342)
+- 2023-10-24: Merged adding a metric required for beta promotion [feat: add job_pods_creation_total metric](https://github.com/kubernetes/kubernetes/pull/121481)
+- 2023-10-27: Merged [Switch feature flag to beta for pod replacement policy and add e2e test #121491](https://github.com/kubernetes/kubernetes/pull/121491)
```

Comment:
I would extend the history with the two PRs which happened in the second Beta:
- https://github.com/kubernetes/kubernetes/pull/125175
- https://github.com/kubernetes/kubernetes/pull/125510

Reply:
I will add a note on this in the implementation history so it is easy to understand that first beta was promoted in v1.29 and then important updates happened in v1.31